### PR TITLE
Debian

### PR DESCRIPTION
**Pull request by jburel**
Add support for debian9

See https://trello.com/c/00CYQi2t/29-debian-installation-page

Tested by travis

### CODE REVIEW

File: ansible/debian-ice3.6/websession_configure.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+set +u
+source /home/omero/omerowebvenv/bin/activate
+set -u
+
+pip install --upgrade -r /home/omero/OMERO.py/share/web/requirements-redis.txt
+
+/home/omero/OMERO.py/bin/omero config set omero.web.session_engine 'django.contrib.sessions.backends.cache'
+/home/omero/OMERO.py/bin/omero config set omero.web.caches '{"default": {"BACKEND": "django_redis.cache.RedisCache","LOCATION": "redis://127.0.0.1:6379/0"}}'
```
Comment (mtbc) [inline ansible/debian-ice3.6/websession_configure.sh]: space after comma? (*not* a blocker)

File: ansible/templates/omeroweb-install-doc.txt.j2 [hunk 2]
```diff
@@ -61,13 +61,13 @@
...
 {% endif %}
-{% if os == 'ubuntu' %}
+{% if os == 'ubuntu' or os == 'debian' %}
 {% set path = os + '/omero-web-init.d.j2'%}
 {% macro style() %}{% include path ignore missing with context %}{% endmacro %}
 {{ style()|indent }}
 {% endif %}
 
-{% if os == 'ubuntu' or os == 'osx' %}
+{% if os == 'ubuntu' or os == 'osx' or  os == 'debian' %}
```
Comment (mtbc) [inline ansible/templates/omeroweb-install-doc.txt.j2]: double space!

Comment (jburel) [inline ansible/templates/omeroweb-install-doc.txt.j2]: did not want to change the pattern already in use in the file

**Comment by jburel** [comment]
@mtbc ok to merge when travis is green?

**Comment by mtbc** [comment]
Yes, no blockers from me!

**Pull request merged**